src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -223,6 +238,17 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
     columns.sort();
     this.columns = columns;
 
+    // add scores columns
```

Comment:
nitpick but may be more elegant
```
for (const name  of  ['GENETIC','MULTI-OMICS','RISK SCORE']) {
 if (!columns.includes(name))
      columns.unshift(name);
}
```

Reply:
I'll refactor this.  Looks like I was using it in two places